client/components/jetpack/sidebar/style.scss

```diff
@@ -3,22 +3,73 @@
 // Menu links
 .sidebar__jetpack-cloud {
 
-	.current-site__switch-sites .button.is-borderless .dashicons-before {
-		top: 15px;
+	.site__home {
+		background-color: var( --color-sidebar-menu-hover-text );
+
+		svg {
+			fill: var( --color-sidebar-background );
+		}
+		
+	}
+
+	.current-site__switch-sites .button.is-borderless {
+		border-radius: 0;
+
+		&:hover {
+			background-color: var( --color-sidebar-menu-hover-background );
+		}
+
+		 .dashicons-before {
+			top: 15px;
+		}
+	}
+
+	.selected {
+
+		.sidebar__menu-link {
+			background-color: var( --color-sidebar-menu-selected-background );
+		}
+
+		.sidebar__menu-link-text {
+			color: var( --color-sidebar-menu-hover-text );
+		}
+
+		.sidebar__menu-icon {
+			fill: var( --color-sidebar-menu-selected-gridicon-fill );
+		}
+
+		&:hover {
+			.sidebar__menu-icon {
+				fill: var( --color-sidebar-menu-selected-gridicon-fill );
+			}
+		}
 	}
 
 	.sidebar__menu-link {
 		padding-top: 14px;
 		padding-bottom: 14px;
 
+		&:hover {
```

Comment:
Since we're touching on this, could we add `focus` styles too?

Reply:
Sorry for the delay. I pushed a new commit adding an active state, it's subtle but I think might work. I tried `:focus` but it didn't work well since the state stayed active after pressing click. Let me know if this is ok.

Reply:
It's good to add `:active` styles but my concern with the `:focus` state was to make sure the sidebar was accessible while being navigated across. For example, when tabbing through the various items, none of them are highlighted because we don't have `:focus` styles.